### ISSUE TITLE
fine tune the pre-trained model on UCF101

### ISSUE THREAD

**Issue by Yueeeeee-1** [moabitcoin/ig65m-pytorch#38]
if I want to fine tune the pre-trained model on UCF101, how could I get the performance of 96.8%?
In my settings, fine-tuning was only performed to train layer4 and the fully connected layer, and the learning rate is 0.0001, am I wrong? Cause the result I got just 78.5%,  can you help me? Thank you!

**Comment by bjuncek** [comment]
Other than parameter fine-tuning (I'd say 15 epochs, LR:1e-4 for the convolutional blocks and 1e-3 for the FC layer), make sure the evaluation script is correct.

If I recall correctly, the results are reported as video level accuracy on 10 uniformly sampled clips per video, averaged over 3 cross validation splits. I'd expect the clip level accuracy (what you're likely measuring) to be around >80%.

**Comment by Yueeeeee-1** [comment]
@bjuncek  Thanks for your reply, I did test the clip level accuracy, and I didn't notice the difference between the video level and clip level,  I will check it.
and for the parameter fine-tuning, ''LR:1e-4 for the convolutional blocks '' , it means all convolutional blocks or just the last one?
thanks for your patience!

**Comment by bjuncek** [comment]
Make sure you're averaging softmaxes rather than predictions (w.r.t. the video level acc).

For all conv blocks

**Comment by paden118** [comment]
Hi, what is the accuracy of your final fine-tuning on ucf101?
